### ISSUE TITLE
text is cutting lower parts of letters

### ISSUE THREAD

**Issue by wizza-smile** [mdgriffith/style-elements#88]
Hi!

When using the text Element, the rendered text is using a HTML <span> with display: block; and overflow: hidden;

This is cutting the lower parts of letters like 'g'. You can see this best on big h1 texts.

I found out that there is a Modify.makeInline function in the Internals, that would render the text properly, but since it is Internal, that does not seem to be the correct way.

Am I the only one with this problem?

## Versions

- Elm: 0.18
- style-elements: 4.0.0

**Comment by mdgriffith** [comment]
Yeah, this should be fixed in 4.1 once I release it.  The manual fix is to bump up the `Font.lineHeight` to something like `1.2`.  The default is `1.0` which apparently isn't big enough to show the entire font. who knew? :/

**Comment by wizza-smile** [comment]
haha1 thanx 👍

**Comment by bengolds** [comment]
Quick note on this -- text that is also wide (for example, italics) gets cut off too, even with the `lineHeight` fix:
https://ellie-app.com/7qxCJtPdYa1/6

Is the overflow: hidden piece strictly necessary?

**Comment by ghpu** [comment]
Still having this problem with 4.2.1
